editor/package.json

```diff
@@ -78,6 +78,7 @@
     "@popperjs/core": "2.4.4",
     "@seznam/compose-react-refs": "^1.0.4",
     "@tippyjs/react": "4.1.0",
+    "@types/eslint": "^7.2.2",
```

Comment:
Interesting... do we need this? If so, presumably this version doesn't match up with the version of eslint we are using (`6.8.0`) - if we do need this please can you match up the types and remove the `^`

Reply:
i updated the version of eslint instead 💁‍♂️